content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -0,0 +1,360 @@
+```
+
+```shell
+? Amount to deposit 5
+? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 
+? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+```
+
+To verify that this worked, let's check the new balance of our address on the L2 chain:
+
+```bash
+zksync-cli wallet balance --address 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 --rpc http://localhost:3050
+```
+
+Note that we aren't checking for any specific token address here.
+The CLI will display the amount as ETH, but in reality, it's our bridged ERC20 tokens.
+
+And that's it! You are now running a local ZK chain that uses a custom ERC20 base token.
+You can follow the steps in the previous section to deploy a contract to the new chain.
+
+## Bridging ETH to your chain
+
+If your ERC20 token bridged to the L2 "looks" like ETH, then what happens if you bridge ETH to your custom chain?
+
+The answer is that if you try to bridge regular ETH to this chain, it will just use a different token contract address.
+
+You can try this out by depositing ETH from the L1 to this new chain:
+
+```bash
+npx zksync-cli bridge deposit --rpc=http://localhost:3050 --l1-rpc=http://localhost:8545
+```
+
+```shell
+? Amount to deposit 20
+? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110 
+? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+```
+
+If you run the previous command to check the base token balance again, note how the balance is still `5`.
+This is because you have to look at the L2 token balance for ETH.
+To find the L2 token address for ETH, you can use the `l2TokenAddress` method available through `zksync-ethers`.
+
+To try this out, open the `zk-chain-test` folder from the previous section,
+and run the commands below to create a new script file:
+
+```bash
+mkdir scripts
+touch scripts/checkBalance.ts
+```
+
+Next, copy and paste the script below to into the `checkBalance.ts` file:
+
+```ts
+import { ETH_ADDRESS_IN_CONTRACTS } from "zksync-ethers/build/utils.js";
+import { getWallet } from "../deploy/utils";
+
+async function main(){
+    const wallet = getWallet();
+    const l2ETHAddress = await wallet.l2TokenAddress(ETH_ADDRESS_IN_CONTRACTS);
+    console.log("L2 ETH Address:", l2ETHAddress);
+
+    const balance = await wallet.getBalance(l2ETHAddress);
+    console.log("L2 ETH Balance 🎉:", balance);
+}
+
+main();
+```
+
+Run the script with the `hardhat run` command:
+
+```bash
+npx hardhat run scripts/checkBalance.ts
+```
+
+You should see the output below showing the same amount of ETH you bridged:
+
+```shell
+L2 ETH Address: 0x_YOUR_ETH_TOKEN_ADDRESS
+L2 ETH Balance 🎉: 20000000000000000000n
+```
+
+## Switching between chains and shutting down the ecosystem
+
+You can switch in between different chains without losing any state by shutting down the chain server and running the command below:
```

Comment:
Does this mean the command below will shut down the chain server, or is there a specific way to do that and the command below is just for switching chains?